### ISSUE TITLE
Trailers Shortcut Stopped Working Correctly

### ISSUE THREAD

**Issue by jeneral** [pix/aeon#92]
I'm not certain when this happened. Late last week the main menu shortcut 'Trailers' was working perfectly now I get blank info panels and I can't select a 'style' for the view:

http://a.imageshack.us/img441/7484/screenshot000z.jpg

My guess is these are cached trailers from when it last worked. If I click info I can see the info and they all play fine if selected.

For testing purposes, I installed a clean XBMC to a new folder and added Aeon65 as well as the AMT - Lite add-on. When clicking on the Trailers shortcut, I get the following (Apple Movie Trailers is installed and works fine if run from Video Addons):

http://a.imageshack.us/img337/7650/screenshot000w.jpg

**Comment by pix** [comment]
Seems to work for me, Anybody else affected by this ?

**Comment by jeneral** [comment]
First off, I don't have Youtube trailers enabled.

The first screenshot is from my XBMC. The second screenshot is from a CLEAN install. Have you tried a clean install? I did find out a glitch. On the clean install, the second screenshot will appear if a video database doesn't exist. Once I added a video source and created a few files, I ended up with a result like the first screenshot. All the covers are there but no info, trigger can't be changed and styles is empty. I also re-downloaded aeon in case that was the problem and it didn't make a difference.

Here is my debug log:

http://pastebin.com/0GMVXpf4

**Comment by pix** [comment]
In order to have a working trailer addon, you need to have some movies in your library, this is why it doesn't work in the new folder, for the first problem, I have not idea. 

I tested with a clean install, once the the movie library has some content everything's working

**Comment by jeneral** [comment]
After your hint (movie library), I tried a few things. My movies are on my NAS so I rarely use it unless I plan to watch a movie. Well today I accessed my movies (Showcase view) and everything was fine. I went back to Trailers and I still had the same thing as the first pic. I went back to movies, updated library and switched the view to Multiplex. When I went back to Trailers, everything was back to normal. I reset both Movies and Trailers to Showcase  and exited XBMC. When I went back in Trailers was messed up again. Repeating that same procedure fixed it and now it appears to have kept the settings.

There is definitely some glitch that is causing views to change (I see it from time to time in music). I just didn't realize Movies and Trailers were linked. At least I know how to 'fix' the issue in the future.
